cmd/kogito/command/shared/mkdir.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package shared
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+// Mkdir creates directory dir, including non-existing parent directories.
+func Mkdir(dir string) {
+	_, err := os.Stat(dir)
+	if os.IsNotExist(err) {
+		errDir := os.MkdirAll(dir, 0755)
+		if errDir != nil {
+			panic(err)
+		}
+	}
+}
+
+// TempDirWithFile creates a temporary directory dir with a
+// temporary file
+func TempDirWithFile(dir string, file string) string {
```

Comment:
Is this function being used only on tests? If it's the case, can you please move to the `test` package where we have the shared test util functions?